### ISSUE TITLE
Error Syncing User When "Reset Everybody" is Used

### ISSUE THREAD

**Issue by tohuw** [warlof/seat-discord-connector#57]
This is the error in the Laravel log:

`[2019-07-09 04:52:09] local.ERROR: Undefined variable: roles [{"file":"/var/www/seatnui/vendor/warlof/seat-discord-connector/src/Jobs/MemberOrchestrator.php","line":211,"function":"handleError","class":"Illuminate\\Foundation\\Bootstrap\\HandleExceptions","type":"->","args":[8,"Undefined variable: roles","/var/www/seatnui/vendor/warlof/seat-discord-connector/src/Jobs/MemberOrchestrator.php",211,{"member":"[object] (RestCord\\Model\\Guild\\GuildMember: {\"deaf\":false,\"joined_at\":{\"date\":\"2019-07-09 04:10:09.652000\",\"timezone_type\":1,\"timezone\":\"+00:00\"},\"mute\":false,\"nick\":\"[NUI.] Sinpwnz0rz\",\"roles\":[597845234067701762],\"user\":{\"avatar\":\"5808a0877974ede54594419a8813e81c\",\"bot\":false,\"discriminator\":\"1930\",\"email\":null,\"flags\":null,\"id\":243954435720478722,\"locale\":null,\"mfa_enabled\":false,\"premium_type\":null,\"username\":\"Sin\",\"verified\":false}})","discord_user":"[object] (Warlof\\Seat\\Connector\\Discord\\Models\\DiscordUser: {\"group_id\":3,\"discord_id\":243954435720478722,\"nick\":\"[NUI.] Sinpwnz0rz\",\"refresh_token\":\"REDACTED\",\"access_token\":\"REDACTED\",\"expires_at\":\"2019-07-16 04:10:09\",\"scope\":\"identify guilds.join\",\"created_at\":\"2019-07-09 04:07:39\",\"updated_at\":\"2019-07-09 04:10:09\",\"group\":{\"id\":3,\"created_at\":\"2019-07-09 04:03:16\",\"updated_at\":\"2019-07-09 04:03:16\",\"main_character\":{\"character_id\":462055792,\"name\":\"Sinpwnz0rz\",\"description\":\"<font size=\\\"12\\\" color=\\\"#bfffffff\\\"><br></font><font size=\\\"12\\\" color=\\\"#ffffffff\\\">Exodus. Alliance Tournament Captain<br><br>[20:32:19] Walter Cohen &gt; Exodus and Camel .. That was a display of skill from both of you that is beyond most of us ... Thank you ... WOW<br><br><br>[03:45:56] </font><font size=\\\"12\\\" color=\\\"#ffd98d00\\\"><loc><a href=\\\"showinfo:1374//826711106\\\">Nusak</a></loc></font><font size=\\\"12\\\" color=\\\"#ffffffff\\\">  </font><font size=\\\"12\\\" color=\\\"#ffd98d00\\\"><a href=\\\"showinfo:1379//462055792\\\">Sinpwnz0rz</a></font><font size=\\\"12\\\" color=\\\"#ffffffff\\\"> Hats off. Good flying<br>[03:10:02] </font><font size=\\\"12\\\" color=\\\"#ffd98d00\\\"><loc><a href=\\\"showinfo:1379//691964524\\\">Norgron</a></loc></font><font size=\\\"12\\\" color=\\\"#ffffffff\\\">  </font><font size=\\\"12\\\" color=\\\"#ffd98d00\\\"><a href=\\\"showinfo:1379//462055792\\\">Sinpwnz0rz</a></font><font size=\\\"12\\\" color=\\\"#ffffffff\\\"> i salute you for still being alive<br>[05:40:26] </font><font size=\\\"12\\\" color=\\\"#ffd98d00\\\"><loc><a href=\\\"showinfo:1378//1230767333\\\">Dark Aiden</a></loc></font><font size=\\\"12\\\" color=\\\"#ffffffff\\\"> &gt; im fairly certain you are a alt tho hakers alt yes?<br>[05:41:11] Dark Aiden &gt; hakers alt yes?<br>[05:41:25] Dark Aiden &gt; you fly like you have been in a ares plenty of time before<br><br>1:12 AM - Hazard: playing FPS with a game pad is like masturbating using the prosthetic hand <br><br>[23:29:43] Knerf &gt; props to cepter<br></font><font size=\\\"24\\\" color=\\\"#ff7f0000\\\">Trust in the rust!<br><br></font><font size=\\\"12\\\" color=\\\"#ff00ff00\\\"><u>The will to help, is always more valuable than the ability to do so.</u></font>\",\"corporation_id\":98591755,\"alliance_id\":99008788,\"birthday\":\"2010-04-28T03:38:00Z\",\"gender\":\"male\",\"race_id\":2,\"bloodline_id\":3,\"ancestry_id\":19,\"security_status\":-5.8,\"faction_id\":null,\"created_at\":\"2019-07-09 04:03:19\",\"updated_at\":\"2019-07-09 04:03:19\"},\"refresh_tokens\":[{\"character_id\":462055792,\"refresh_token\":\"REDACTED\",\"scopes\":[\"esi-calendar.read_calendar_events.v1\",\"esi-location.read_location.v1\",\"esi-location.read_ship_type.v1\",\"esi-mail.read_mail.v1\",\"esi-skills.read_skills.v1\",\"esi-skills.read_skillqueue.v1\",\"esi-wallet.read_character_wallet.v1\",\"esi-search.search_structures.v1\",\"esi-clones.read_clones.v1\",\"esi-characters.read_contacts.v1\",\"esi-universe.read_structures.v1\",\"esi-bookmarks.read_character_bookmarks.v1\",\"esi-killmails.read_killmails.v1\",\"esi-corporations.read_corporation_membership.v1\",\"esi-assets.read_assets.v1\",\"esi-planets.manage_planets.v1\",\"esi-fleets.read_fleet.v1\",\"esi-ui.open_window.v1\",\"esi-fittings.read_fittings.v1\",\"esi-markets.structure_markets.v1\",\"esi-corporations.read_structures.v1\",\"esi-characters.read_loyalty.v1\",\"esi-characters.read_opportunities.v1\",\"esi-characters.read_chat_channels.v1\",\"esi-characters.read_medals.v1\",\"esi-characters.read_standings.v1\",\"esi-characters.read_agents_research.v1\",\"esi-industry.read_character_jobs.v1\",\"esi-markets.read_character_orders.v1\",\"esi-characters.read_blueprints.v1\",\"esi-characters.read_corporation_roles.v1\",\"esi-location.read_online.v1\",\"esi-contracts.read_character_contracts.v1\",\"esi-clones.read_implants.v1\",\"esi-characters.read_fatigue.v1\",\"esi-killmails.read_corporation_killmails.v1\",\"esi-corporations.track_members.v1\",\"esi-wallet.read_corporation_wallets.v1\",\"esi-characters.read_notifications.v1\",\"esi-corporations.read_divisions.v1\",\"esi-corporations.read_contacts.v1\",\"esi-assets.read_corporation_assets.v1\",\"esi-corporations.read_titles.v1\",\"esi-corporations.read_blueprints.v1\",\"esi-bookmarks.read_corporation_bookmarks.v1\",\"esi-contracts.read_corporation_contracts.v1\",\"esi-corporations.read_standings.v1\",\"esi-corporations.read_starbases.v1\",\"esi-industry.read_corporation_jobs.v1\",\"esi-markets.read_corporation_orders.v1\",\"esi-corporations.read_container_logs.v1\",\"esi-industry.read_character_mining.v1\",\"esi-industry.read_corporation_mining.v1\",\"esi-planets.read_customs_offices.v1\",\"esi-corporations.read_facilities.v1\",\"esi-corporations.read_medals.v1\",\"esi-characters.read_titles.v1\",\"esi-alliances.read_contacts.v1\",\"esi-characters.read_fw_stats.v1\",\"esi-corporations.read_fw_stats.v1\",\"esi-characterstats.read.v1\"],\"expires_on\":\"2019-07-09 04:23:15\",\"token\":null,\"created_at\":\"2019-07-09 04:03:16\",\"updated_at\":\"2019-07-09 04:03:16\",\"deleted_at\":null,\"group_id\":3}],\"users\":[{\"id\":462055792,\"group_id\":3,\"name\":\"Sinpwnz0rz\",\"active\":true,\"character_owner_hash\":\"hMFJBeORG7jKJ/v4dwKsH0nmhQY=\",\"last_login\":\"2019-07-09 04:03:16\",\"last_login_source\":\"50.93.36.228\",\"created_at\":\"2019-07-09 04:03:16\",\"updated_at\":\"2019-07-09 04:03:16\",\"email\":\"\"}]}})","new_nickname":null,"pending_drops":"[object] (Illuminate\\Support\\Collection: [597845234067701762])","pending_adds":"[object] (Illuminate\\Support\\Collection: [])","expected_nickname":"[NUI.] Sinpwnz0rz","role_id":597845234067701762,"is_roles_outdated":true}]},{"file":"/var/www/seatnui/vendor/warlof/seat-discord-connector/src/Jobs/MemberOrchestrator.php","line":155,"function":"processMappingBase","class":"Warlof\\Seat\\Connector\\Discord\\Jobs\\MemberOrchestrator","type":"->","args":["[object] (RestCord\\Model\\Guild\\GuildMember: {\"deaf\":false,\"joined_at\":{\"date\":\"2019-07-09 04:10:09.652000\",\"timezone_type\":1,\"timezone\":\"+00:00\"},\"mute\":false,\"nick\":\"[NUI.] Sinpwnz0rz\",\"roles\":[597845234067701762],\"user\":{\"avatar\":\"5808a0877974ede54594419a8813e81c\",\"bot\":false,\"discriminator\":\"1930\",\"email\":null,\"flags\":null,\"id\":243954435720478722,\"locale\":null,\"mfa_enabled\":false,\"premium_type\":null,\"username\":\"Sin\",\"verified\":false}})","[object] (Warlof\\Seat\\Connector\\Discord\\Models\\DiscordUser: {\"group_id\":3,\"discord_id\":243954435720478722,\"nick\":\"[NUI.] Sinpwnz0rz\",\"refresh_token\":\"REDACTED\",\"access_token\":\"REDACTED\",\"expires_at\":\"2019-07-16 04:10:09\",\"scope\":\"identify guilds.join\",\"created_at\":\"2019-07-09 04:07:39\",\"updated_at\":\"2019-07-09 04:10:09\",\"group\":{\"id\":3,\"created_at\":\"2019-07-09 04:03:16\",\"updated_at\":\"2019-07-09 04:03:16\",\"main_character\":{\"character_id\":462055792,\"name\":\"Sinpwnz0rz\",\"description\":\"<font size=\\\"12\\\" color=\\\"#bfffffff\\\"><br></font><font size=\\\"12\\\" color=\\\"#ffffffff\\\">Exodus. Alliance Tournament Captain<br><br>[20:32:19] Walter Cohen &gt; Exodus and Camel .. That was a display of skill from both of you that is beyond most of us ... Thank you ... WOW<br><br><br>[03:45:56] </font><font size=\\\"12\\\" color=\\\"#ffd98d00\\\"><loc><a href=\\\"showinfo:1374//826711106\\\">Nusak</a></loc></font><font size=\\\"12\\\" color=\\\"#ffffffff\\\">  </font><font size=\\\"12\\\" color=\\\"#ffd98d00\\\"><a href=\\\"showinfo:1379//462055792\\\">Sinpwnz0rz</a></font><font size=\\\"12\\\" color=\\\"#ffffffff\\\"> Hats off. Good flying<br>[03:10:02] </font><font size=\\\"12\\\" color=\\\"#ffd98d00\\\"><loc><a href=\\\"showinfo:1379//691964524\\\">Norgron</a></loc></font><font size=\\\"12\\\" color=\\\"#ffffffff\\\">  </font><font size=\\\"12\\\" color=\\\"#ffd98d00\\\"><a href=\\\"showinfo:1379//462055792\\\">Sinpwnz0rz</a></font><font size=\\\"12\\\" color=\\\"#ffffffff\\\"> i salute you for still being alive<br>[05:40:26] </font><font size=\\\"12\\\" color=\\\"#ffd98d00\\\"><loc><a href=\\\"showinfo:1378//1230767333\\\">Dark Aiden</a></loc></font><font size=\\\"12\\\" color=\\\"#ffffffff\\\"> &gt; im fairly certain you are a alt tho hakers alt yes?<br>[05:41:11] Dark Aiden &gt; hakers alt yes?<br>[05:41:25] Dark Aiden &gt; you fly like you have been in a ares plenty of time before<br><br>1:12 AM - Hazard: playing FPS with a game pad is like masturbating using the prosthetic hand <br><br>[23:29:43] Knerf &gt; props to cepter<br></font><font size=\\\"24\\\" color=\\\"#ff7f0000\\\">Trust in the rust!<br><br></font><font size=\\\"12\\\" color=\\\"#ff00ff00\\\"><u>The will to help, is always more valuable than the ability to do so.</u></font>\",\"corporation_id\":98591755,\"alliance_id\":99008788,\"birthday\":\"2010-04-28T03:38:00Z\",\"gender\":\"male\",\"race_id\":2,\"bloodline_id\":3,\"ancestry_id\":19,\"security_status\":-5.8,\"faction_id\":null,\"created_at\":\"2019-07-09 04:03:19\",\"updated_at\":\"2019-07-09 04:03:19\"},\"refresh_tokens\":[{\"character_id\":462055792,\"refresh_token\":\"REDACTED\",\"scopes\":[\"esi-calendar.read_calendar_events.v1\",\"esi-location.read_location.v1\",\"esi-location.read_ship_type.v1\",\"esi-mail.read_mail.v1\",\"esi-skills.read_skills.v1\",\"esi-skills.read_skillqueue.v1\",\"esi-wallet.read_character_wallet.v1\",\"esi-search.search_structures.v1\",\"esi-clones.read_clones.v1\",\"esi-characters.read_contacts.v1\",\"esi-universe.read_structures.v1\",\"esi-bookmarks.read_character_bookmarks.v1\",\"esi-killmails.read_killmails.v1\",\"esi-corporations.read_corporation_membership.v1\",\"esi-assets.read_assets.v1\",\"esi-planets.manage_planets.v1\",\"esi-fleets.read_fleet.v1\",\"esi-ui.open_window.v1\",\"esi-fittings.read_fittings.v1\",\"esi-markets.structure_markets.v1\",\"esi-corporations.read_structures.v1\",\"esi-characters.read_loyalty.v1\",\"esi-characters.read_opportunities.v1\",\"esi-characters.read_chat_channels.v1\",\"esi-characters.read_medals.v1\",\"esi-characters.read_standings.v1\",\"esi-characters.read_agents_research.v1\",\"esi-industry.read_character_jobs.v1\",\"esi-markets.read_character_orders.v1\",\"esi-characters.read_blueprints.v1\",\"esi-characters.read_corporation_roles.v1\",\"esi-location.read_online.v1\",\"esi-contracts.read_character_contracts.v1\",\"esi-clones.read_implants.v1\",\"esi-characters.read_fatigue.v1\",\"esi-killmails.read_corporation_killmails.v1\",\"esi-corporations.track_members.v1\",\"esi-wallet.read_corporation_wallets.v1\",\"esi-characters.read_notifications.v1\",\"esi-corporations.read_divisions.v1\",\"esi-corporations.read_contacts.v1\",\"esi-assets.read_corporation_assets.v1\",\"esi-corporations.read_titles.v1\",\"esi-corporations.read_blueprints.v1\",\"esi-bookmarks.read_corporation_bookmarks.v1\",\"esi-contracts.read_corporation_contracts.v1\",\"esi-corporations.read_standings.v1\",\"esi-corporations.read_starbases.v1\",\"esi-industry.read_corporation_jobs.v1\",\"esi-markets.read_corporation_orders.v1\",\"esi-corporations.read_container_logs.v1\",\"esi-industry.read_character_mining.v1\",\"esi-industry.read_corporation_mining.v1\",\"esi-planets.read_customs_offices.v1\",\"esi-corporations.read_facilities.v1\",\"esi-corporations.read_medals.v1\",\"esi-characters.read_titles.v1\",\"esi-alliances.read_contacts.v1\",\"esi-characters.read_fw_stats.v1\",\"esi-corporations.read_fw_stats.v1\",\"esi-characterstats.read.v1\"],\"expires_on\":\"2019-07-09 04:23:15\",\"token\":null,\"created_at\":\"2019-07-09 04:03:16\",\"updated_at\":\"2019-07-09 04:03:16\",\"deleted_at\":null,\"group_id\":3}],\"users\":[{\"id\":462055792,\"group_id\":3,\"name\":\"Sinpwnz0rz\",\"active\":true,\"character_owner_hash\":\"hMFJBeORG7jKJ/v4dwKsH0nmhQY=\",\"last_login\":\"2019-07-09 04:03:16\",\"last_login_source\":\"50.93.36.228\",\"created_at\":\"2019-07-09 04:03:16\",\"updated_at\":\"2019-07-09 04:03:16\",\"email\":\"\"}]}})"]},{"function":"handle","class":"Warlof\\Seat\\Connector\\Discord\\Jobs\\MemberOrchestrator","type":"->","args":[]},{"file":"/var/www/seatnui/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php","line":29,"function":"call_user_func_array","args":[["[object] (Warlof\\Seat\\Connector\\Discord\\Jobs\\MemberOrchestrator: {\"tries\":100,\"connection\":null,\"queue\":\"high\",\"chainConnection\":null,\"chainQueue\":null,\"delay\":null,\"chained\":[]})","handle"],[]]},{"file":"/var/www/seatnui/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php","line":87,"function":"Illuminate\\Container\\{closure}","class":"Illuminate\\Container\\BoundMethod","type":"::","args":[]},{"file":"/var/www/seatnui/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php","line":31,"function":"callBoundMethod","class":"Illuminate\\Container\\BoundMethod","type":"::","args":["[object] (Illuminate\\Foundation\\Application: {\"contextual\":[]})",["[object] (Warlof\\Seat\\Connector\\Discord\\Jobs\\MemberOrchestrator: {\"tries\":100,\"connection\":null,\"queue\":\"high\",\"chainConnection\":null,\"chainQueue\":null,\"delay\":null,\"chained\":[]})","handle"],"[object] (Closure: {})"]},{"file":"/var/www/seatnui/vendor/laravel/framework/src/Illuminate/Container/Container.php","line":549,"function":"call","class":"Illuminate\\Container\\BoundMethod","type":"::","args":["[object] (Illuminate\\Foundation\\Application: {\"contextual\":[]})",["[object] (Warlof\\Seat\\Connector\\Discord\\Jobs\\MemberOrchestrator: {\"tries\":100,\"connection\":null,\"queue\":\"high\",\"chainConnection\":null,\"chainQueue\":null,\"delay\":null,\"chained\":[]})","handle"],[],null]},{"file":"/var/www/seatnui/vendor/laravel/framework/src/Illuminate/Bus/Dispatcher.php","line":94,"function":"call","class":"Illuminate\\Container\\Container","type":"->","args":[["[object] (Warlof\\Seat\\Connector\\Discord\\Jobs\\MemberOrchestrator: {\"tries\":100,\"connection\":null,\"queue\":\"high\",\"chainConnection\":null,\"chainQueue\":null,\"delay\":null,\"chained\":[]})","handle"]]},{"file":"/var/www/seatnui/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php","line":114,"function":"Illuminate\\Bus\\{closure}","class":"Illuminate\\Bus\\Dispatcher","type":"->","args":["[object] (Warlof\\Seat\\Connector\\Discord\\Jobs\\MemberOrchestrator: {\"tries\":100,\"connection\":null,\"queue\":\"high\",\"chainConnection\":null,\"chainQueue\":null,\"delay\":null,\"chained\":[]})"]},{"file":"/var/www/seatnui/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php","line":102,"function":"Illuminate\\Pipeline\\{closure}","class":"Illuminate\\Pipeline\\Pipeline","type":"->","args":["[object] (Warlof\\Seat\\Connector\\Discord\\Jobs\\MemberOrchestrator: {\"tries\":100,\"connection\":null,\"queue\":\"high\",\"chainConnection\":null,\"chainQueue\":null,\"delay\":null,\"chained\":[]})"]},{"file":"/var/www/seatnui/vendor/laravel/framework/src/Illuminate/Bus/Dispatcher.php","line":98,"function":"then","class":"Illuminate\\Pipeline\\Pipeline","type":"->","args":["[object] (Closure: {})"]},{"file":"/var/www/seatnui/vendor/laravel/framework/src/Illuminate/Queue/CallQueuedHandler.php","line":49,"function":"dispatchNow","class":"Illuminate\\Bus\\Dispatcher","type":"->","args":["[object] (Warlof\\Seat\\Connector\\Discord\\Jobs\\MemberOrchestrator: {\"tries\":100,\"connection\":null,\"queue\":\"high\",\"chainConnection\":null,\"chainQueue\":null,\"delay\":null,\"chained\":[]})",false]},{"file":"/var/www/seatnui/vendor/laravel/framework/src/Illuminate/Queue/Jobs/Job.php","line":76,"function":"call","class":"Illuminate\\Queue\\CallQueuedHandler","type":"->","args":["[object] (Illuminate\\Queue\\Jobs\\RedisJob: {})",{"commandName":"Warlof\\Seat\\Connector\\Discord\\Jobs\\MemberOrchestrator","command":"O:53:\"Warlof\\Seat\\Connector\\Discord\\Jobs\\MemberOrchestrator\":14:{s:7:\"\u0000*\u0000tags\";a:2:{i:0;s:12:\"orchestrator\";i:1;s:10:\"terminator\";}s:61:\"\u0000Warlof\\Seat\\Connector\\Discord\\Jobs\\MemberOrchestrator\u0000client\";N;s:60:\"\u0000Warlof\\Seat\\Connector\\Discord\\Jobs\\MemberOrchestrator\u0000guild\";N;s:62:\"\u0000Warlof\\Seat\\Connector\\Discord\\Jobs\\MemberOrchestrator\u0000members\";N;s:60:\"\u0000Warlof\\Seat\\Connector\\Discord\\Jobs\\MemberOrchestrator\u0000roles\";N;s:65:\"\u0000Warlof\\Seat\\Connector\\Discord\\Jobs\\MemberOrchestrator\u0000terminator\";b:1;s:5:\"tries\";i:100;s:6:\"\u0000*\u0000job\";N;s:10:\"connection\";N;s:5:\"queue\";s:4:\"high\";s:15:\"chainConnection\";N;s:10:\"chainQueue\";N;s:5:\"delay\";N;s:7:\"chained\";a:0:{}}"}]},{"file":"/var/www/seatnui/vendor/laravel/framework/src/Illuminate/Queue/Worker.php","line":320,"function":"fire","class":"Illuminate\\Queue\\Jobs\\Job","type":"->","args":[]},{"file":"/var/www/seatnui/vendor/laravel/framework/src/Illuminate/Queue/Worker.php","line":270,"function":"process","class":"Illuminate\\Queue\\Worker","type":"->","args":["redis","[object] (Illuminate\\Queue\\Jobs\\RedisJob: {})","[object] (Illuminate\\Queue\\WorkerOptions: {\"delay\":\"0\",\"memory\":\"128\",\"timeout\":\"900\",\"sleep\":\"3\",\"maxTries\":\"1\",\"force\":false})"]},{"file":"/var/www/seatnui/vendor/laravel/framework/src/Illuminate/Queue/Worker.php","line":114,"function":"runJob","class":"Illuminate\\Queue\\Worker","type":"->","args":["[object] (Illuminate\\Queue\\Jobs\\RedisJob: {})","redis","[object] (Illuminate\\Queue\\WorkerOptions: {\"delay\":\"0\",\"memory\":\"128\",\"timeout\":\"900\",\"sleep\":\"3\",\"maxTries\":\"1\",\"force\":false})"]},{"file":"/var/www/seatnui/vendor/laravel/framework/src/Illuminate/Queue/Console/WorkCommand.php","line":101,"function":"daemon","class":"Illuminate\\Queue\\Worker","type":"->","args":["redis","high,medium,low,default","[object] (Illuminate\\Queue\\WorkerOptions: {\"delay\":\"0\",\"memory\":\"128\",\"timeout\":\"900\",\"sleep\":\"3\",\"maxTries\":\"1\",\"force\":false})"]},{"file":"/var/www/seatnui/vendor/laravel/framework/src/Illuminate/Queue/Console/WorkCommand.php","line":85,"function":"runWorker","class":"Illuminate\\Queue\\Console\\WorkCommand","type":"->","args":["redis","high,medium,low,default"]},{"file":"/var/www/seatnui/vendor/laravel/horizon/src/Console/WorkCommand.php","line":46,"function":"handle","class":"Illuminate\\Queue\\Console\\WorkCommand","type":"->","args":[]},{"function":"handle","class":"Laravel\\Horizon\\Console\\WorkCommand","type":"->","args":[]},{"file":"/var/www/seatnui/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php","line":29,"function":"call_user_func_array","args":[["[object] (Laravel\\Horizon\\Console\\WorkCommand: {})","handle"],[]]},{"file":"/var/www/seatnui/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php","line":87,"function":"Illuminate\\Container\\{closure}","class":"Illuminate\\Container\\BoundMethod","type":"::","args":[]},{"file":"/var/www/seatnui/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php","line":31,"function":"callBoundMethod","class":"Illuminate\\Container\\BoundMethod","type":"::","args":["[object] (Illuminate\\Foundation\\Application: {\"contextual\":[]})",["[object] (Laravel\\Horizon\\Console\\WorkCommand: {})","handle"],"[object] (Closure: {})"]},{"file":"/var/www/seatnui/vendor/laravel/framework/src/Illuminate/Container/Container.php","line":549,"function":"call","class":"Illuminate\\Container\\BoundMethod","type":"::","args":["[object] (Illuminate\\Foundation\\Application: {\"contextual\":[]})",["[object] (Laravel\\Horizon\\Console\\WorkCommand: {})","handle"],[],null]},{"file":"/var/www/seatnui/vendor/laravel/framework/src/Illuminate/Console/Command.php","line":183,"function":"call","class":"Illuminate\\Container\\Container","type":"->","args":[["[object] (Laravel\\Horizon\\Console\\WorkCommand: {})","handle"]]},{"file":"/var/www/seatnui/vendor/symfony/console/Command/Command.php","line":255,"function":"execute","class":"Illuminate\\Console\\Command","type":"->","args":["[object] (Symfony\\Component\\Console\\Input\\ArgvInput: 'horizon:work' redis --delay=0 --memory=128 --queue='high,medium,low,default' --sleep=3 --timeout=900 --tries=1 --supervisor='syndoulos-qfdP:seat-workers')","[object] (Illuminate\\Console\\OutputStyle: {})"]},{"file":"/var/www/seatnui/vendor/laravel/framework/src/Illuminate/Console/Command.php","line":170,"function":"run","class":"Symfony\\Component\\Console\\Command\\Command","type":"->","args":["[object] (Symfony\\Component\\Console\\Input\\ArgvInput: 'horizon:work' redis --delay=0 --memory=128 --queue='high,medium,low,default' --sleep=3 --timeout=900 --tries=1 --supervisor='syndoulos-qfdP:seat-workers')","[object] (Illuminate\\Console\\OutputStyle: {})"]},{"file":"/var/www/seatnui/vendor/symfony/console/Application.php","line":969,"function":"run","class":"Illuminate\\Console\\Command","type":"->","args":["[object] (Symfony\\Component\\Console\\Input\\ArgvInput: 'horizon:work' redis --delay=0 --memory=128 --queue='high,medium,low,default' --sleep=3 --timeout=900 --tries=1 --supervisor='syndoulos-qfdP:seat-workers')","[object] (Symfony\\Component\\Console\\Output\\ConsoleOutput: {})"]},{"file":"/var/www/seatnui/vendor/symfony/console/Application.php","line":255,"function":"doRunCommand","class":"Symfony\\Component\\Console\\Application","type":"->","args":["[object] (Laravel\\Horizon\\Console\\WorkCommand: {})","[object] (Symfony\\Component\\Console\\Input\\ArgvInput: 'horizon:work' redis --delay=0 --memory=128 --queue='high,medium,low,default' --sleep=3 --timeout=900 --tries=1 --supervisor='syndoulos-qfdP:seat-workers')","[object] (Symfony\\Component\\Console\\Output\\ConsoleOutput: {})"]},{"file":"/var/www/seatnui/vendor/symfony/console/Application.php","line":148,"function":"doRun","class":"Symfony\\Component\\Console\\Application","type":"->","args":["[object] (Symfony\\Component\\Console\\Input\\ArgvInput: 'horizon:work' redis --delay=0 --memory=128 --queue='high,medium,low,default' --sleep=3 --timeout=900 --tries=1 --supervisor='syndoulos-qfdP:seat-workers')","[object] (Symfony\\Component\\Console\\Output\\ConsoleOutput: {})"]},{"file":"/var/www/seatnui/vendor/laravel/framework/src/Illuminate/Console/Application.php","line":88,"function":"run","class":"Symfony\\Component\\Console\\Application","type":"->","args":["[object] (Symfony\\Component\\Console\\Input\\ArgvInput: 'horizon:work' redis --delay=0 --memory=128 --queue='high,medium,low,default' --sleep=3 --timeout=900 --tries=1 --supervisor='syndoulos-qfdP:seat-workers')","[object] (Symfony\\Component\\Console\\Output\\ConsoleOutput: {})"]},{"file":"/var/www/seatnui/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php","line":121,"function":"run","class":"Illuminate\\Console\\Application","type":"->","args":["[object] (Symfony\\Component\\Console\\Input\\ArgvInput: 'horizon:work' redis --delay=0 --memory=128 --queue='high,medium,low,default' --sleep=3 --timeout=900 --tries=1 --supervisor='syndoulos-qfdP:seat-workers')","[object] (Symfony\\Component\\Console\\Output\\ConsoleOutput: {})"]},{"file":"/var/www/seatnui/artisan","line":37,"function":"handle","class":"Illuminate\\Foundation\\Console\\Kernel","type":"->","args":["[object] (Symfony\\Component\\Console\\Input\\ArgvInput: 'horizon:work' redis --delay=0 --memory=128 --queue='high,medium,low,default' --sleep=3 --timeout=900 --tries=1 --supervisor='syndoulos-qfdP:seat-workers')","[object] (Symfony\\Component\\Console\\Output\\ConsoleOutput: {})"]}]`

SeAT Info:

```
| Vendor          | Package Name           | Installed Version |
| --------------- | ---------------------- | ----------------- |
| eveseat         | api                    | 3.0.7             |
| eveseat         | console                | 3.0.5             |
| eveseat         | eveapi                 | 3.0.15            |
| eveseat         | notifications          | 3.0.5             |
| eveseat         | services               | 3.0.8             |
| eveseat         | web                    | 3.0.21            |
| herpaderpaldent | seat-groups            | 1.7.2             |
| warlof          | seat-discord-connector | 3.3.1             |
```

**Comment by tohuw** [comment]
Note this happens with every user, it seems.

**Comment by warlof** [comment]
Fixed with 3.3.2
